packages/opentelemetry-exporter-stackdriver-trace/src/transform.ts

```diff
@@ -0,0 +1,128 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { hrTimeToTimeStamp, VERSION } from '@opentelemetry/core';
+import { ReadableSpan } from '@opentelemetry/tracing';
+import * as ot from '@opentelemetry/types';
+import {
+  AttributeMap,
+  Attributes,
+  AttributeValue,
+  Link,
+  LinkType,
+  Span,
+  TruncatableString,
+} from './types';
+
+const AGENT_LABEL_KEY = 'g.co/agent';
```

Comment:
Take from opencensus. Is this something we should keep @mayurkale22? Does the name have any special significance? Could not find any documentation that any attribute keys are reserved or have special meaning.

Reply:
We do this to track the usage of the libraries with Stackdriver, similar label in Go implementation: https://github.com/open-telemetry/opentelemetry-go/blob/09ae5378b779f2bc8b1aa401a9e321b1e9aaf6aa/exporter/trace/stackdriver/trace_proto.go#L40. The value of the label should be in the format of `opentelemetry-js ${LIB_VERSION}; stackdriver-exporter ${EXPORTER_VERSION}`

Reply:
ok i'll update it